api/profiles/dev/application.properties

```diff
@@ -25,6 +25,7 @@ database.initial.pool.size=5
 
 #monitoring configuration
 monitoring.backend=elastic
+monitoring.stats.export.xls.template=${CP_API_MONITORING_XLS_TEMPLATE_PATH:}
```

Comment:
Also could you please add the monitoring excel template file to the repo? And unpack it during api deployment in `deploy/docker/cp-api-srv/init-api`.

Reply:
Done